rust/processor/src/db/common/models/user_transactions_models/parquet_signatures.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(clippy::extra_unused_lifetimes)]
+
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, FieldCount, Serialize)]
+pub struct Signature {
```

Comment:
are you willing to refactor this or just lift and ship. there are a few issues with this and also representation problems (like index not being parsed properly)

- transaction_block_height -> block_height
- type_ -> sig_type
- missing block_timestamp

for improvements, i'd look at this table as example `bigquery-public-data.crypto_aptos_mainnet_us.signatures` with special cases listed here https://www.notion.so/aptoslabs/Debugging-signature-types-2fae12b549604229923505cd99baea46